Gordon360/ApiControllers/HousingController.cs

```diff
@@ -232,17 +242,17 @@ public IHttpActionResult EditApplication([FromBody] ApartmentAppViewModel apartm
         /// change the editor (i.e. primary applicant) of the application
         /// </summary>
         /// <returns>The result of changing the editor</returns>
-        [HttpPost]
-        [Route("apartment/change-editor")]
-        public IHttpActionResult ChangeEditor([FromBody] ApartmentAppNewEditorViewModel newEditorDetails)
+        [HttpPut]
+        [Route("apartment/applications/{applicationID}/{editorUsername}")]
```

Comment:
We actually should not do that, by design.

Reply:
Oh, yeah, that makes sense. I am just thinking ahead: we may want routes that just add/delete/update a single applicant or just add/delete/update a single hall choice so testing is easier. Those can have different Routes though: `[HttpPut] [Route("applicant/{username}")]` and  `...Route("hallchoice...` maybe

Reply:
That is one where the "easier for testing" is sort-of contradicting the design best-practices, so I can see how it could become a bit of a contentious topic.

Reply:
Also, you would want to then use the RESTful naming scheme
`[HttpPut] [Route("apartment/applications/{applicationID}/applicants/{username}")]`
and similarly 
`[HttpPut] [Route("apartment/applications/{applicationID}/halls/{hallname}")]`

Which points out that I should actually change this ones to 
`[HttpPut] [Route("apartment/applications/{applicationID}/editor")]` with the new editor username being sent in the body...